File: Wordplay/python/3.py
# What are all of the words containing a Q but not a U?
def main():
    in_file = open("../sowpods.txt", "r")
    for words in in_file:
        if("Q" in words and "U" not in words):
            print(words, end="")
    
if __name__ == '__main__':
    main()

File: Wordplay/js/3.js
// What are all of the words containing a Q but not a U?

let main = () => {
    
    const fs = require("fs");
    let buffer_arry = fs.readFileSync("Wordplay/sowpods.txt", "utf-8");
    let in_file = buffer_arry.split("\n");

    for(let i = 0; i < in_file.length; i++) {
        if(in_file[i].includes("Q") == true && 
        in_file[i].includes("U") == false) {
            console.log(in_file[i]);
        };
    };
};

main();
